packages/widgets/src/health-monitoring/index.ts

```diff
@@ -19,7 +19,7 @@ export const { definition, componentLoader } = createWidgetDefinition("healthMon
       defaultValue: true,
     }),
   })),
-  supportedIntegrations: ["openmediavault"],
+  supportedIntegrations: ["openmediavault", "getDashDot"],
```

Comment:
```suggestion
  supportedIntegrations: getIntegrationKindsByCategory("healthMonitoring"),
```

Same as in ./packages/api/src/router/widgets/health-monitoring.ts